frontend/src/components/core/AppView/AppView.tsx

```diff
@@ -114,6 +114,7 @@ function AppView(props: AppViewProps): ReactElement {
         uploadClient={uploadClient}
         componentRegistry={componentRegistry}
         formsData={formsData}
+        main={node === elements.main}
```

Comment:
Any reason it has to be in this vertical block? If you want it in the Main area, why not just add it on line 148 after the `renderBlock(elements.main)`?

Reply:
Ah I see it under the hood there's some weirdness. I think having such a solution would be useful, but I'll need to play around with it. It's possible to just put it under the main block for now.